renovate.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "automergeStrategy": "rebase",
+    "automergeType": "pr",
+    "gomod": {
+        "enabled": false
+    },
+    "packageRules": [
```

Comment:
If you don't have the regex section you probably don't need the packageRules either